execution/executor/src/chunk_executor/transaction_chunk.rs

```diff
@@ -76,13 +77,21 @@ impl TransactionChunk for ChunkToExecute {
             })
         };
 
-        let _timer = VM_EXECUTE_CHUNK.start_timer();
-        DoGetExecutionOutput::by_transaction_execution::<V>(
+        let global_cache_manager = GlobalCacheManager::new_with_default_config();
+        global_cache_manager.mark_block_execution_start(&state_view, None)?;
+
+        let timer = VM_EXECUTE_CHUNK.start_timer();
+        let result = DoGetExecutionOutput::by_transaction_execution::<V>(
             sig_verified_txns.into(),
             state_view,
+            &global_cache_manager,
             BlockExecutorConfigFromOnchain::new_no_block_limit(),
             None,
-        )
+        );
+        drop(timer);
```

Comment:
either use a scope or leave it alone.
(Also, the timer used to cover any potential costly drops at the end of the block, for example if `mark_block_execution_end()` returns a thing that's heavy to drop this timer covers it.)

Reply:
Not affected anymore